buildSrc/opensearch-hadoop-version.properties

```diff
@@ -1,4 +1,4 @@
-opensearch_hadoop        = 1.1.0
-opensearch      = 2.12.0
-lucene          = 9.9.2
-build-tools     = 2.11.1
+opensearch_hadoop   = 1.2.0
+opensearch          = 2.12.0
+lucene              = 9.9.2
+build-tools         = 2.11.1
```

Comment:
Can we update OS and build tools to 2.13 and lucene to 9.10?

Reply:
Fixed 👍